### DISCUSSION TITLE
Make portraits disapear

### DISCUSSION THREAD

**Discussion by Santiagodval** [dialogic-godot/dialogic#992] [Q&A] [answered]
So I came up with a solution by changing the dialog box->portraits dim color to a 00 on the A of the rgba, but I was wondering if there is any other way to have more than one character on the same position and make the others disapear instead of just turning slightly grey

**Comment by ii4y-studios** [comment, marked as answer]
the Single Portrait Mode option of a theme should do what you want. Join everyone into the same position, set it to Single Portrait Mode, and only the one talking will actually show up 

![image](https://user-images.githubusercontent.com/2214793/178085854-8c2f7e9d-6f2b-4e1c-8867-42298c04d999.png)

**Comment by zaknafean** [comment]
In addition you can simply make characters 'leave' via the character event, though making people join and leave repeatedly is pretty annoying for a long timeline.
